com.unity.render-pipelines.universal/Editor/UniversalRenderPipelineMaterialUpgrader.cs

```diff
@@ -391,9 +392,22 @@ public static void UpdateStandardSpecularMaterialKeywords(Material material)
             CoreUtils.SetKeyword(material, "_METALLICSPECGLOSSMAP", material.GetTexture("_SpecGlossMap"));
             CoreUtils.SetKeyword(material, "_SPECULAR_SETUP", true);
             UpdateSurfaceTypeAndBlendMode(material);
+            UpdateDetailScaleOffset(material);
             BaseShaderGUI.SetupMaterialBlendMode(material);
         }
 
+        static void UpdateDetailScaleOffset(Material material)
```

Comment:
minor: it would be good to explain the upgrade behavior in comment.

Reply:
Added more info